tesseract/tesseract_planning/tesseract_process_managers/src/process_generators/iterative_spline_parameterization_process_generator.cpp

```diff
@@ -30,10 +30,21 @@ TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 #include <tesseract_process_managers/process_generators/iterative_spline_parameterization_process_generator.h>
-#include <tesseract_command_language/utils/utils.h>
+#include <tesseract_command_language/composite_instruction.h>
+#include <tesseract_command_language/move_instruction.h>
+#include <tesseract_command_language/utils/filter_functions.h>
+#include <tesseract_command_language/utils/flatten_utils.h>
+#include <tesseract_time_parameterization/iterative_spline_parameterization.h>
```

Comment:
Perhaps we should add the iwyu macro to utils.h so you don't have to add all of these

Reply:
Sorry, I am not following what you are suggesting.